main.go

```diff
@@ -2,20 +2,31 @@ package main
 
 import (
 	"errors"
+	"flag"
 	"net/http"
 	"os"
 
 	"code.cloudfoundry.org/lager"
 	"github.com/pivotal-cf/brokerapi"
 )
 
+var strUser = flag.String("user", "", "User name")
+var strPass = flag.String("pass", "", "Password")
+
 func makeLogger() lager.Logger {
 	logger := lager.NewLogger("aws-account-broker")
 	logger.RegisterSink(lager.NewWriterSink(os.Stdout, lager.DEBUG))
 	logger.RegisterSink(lager.NewWriterSink(os.Stderr, lager.ERROR))
 	return logger
 }
 
+func init() {
```

Comment:
This function doesn't appear to be used/useful.

Reply:
Heh, today I learned `init()` is a magic function in Go.

https://golang.org/doc/effective_go.html#init